overlord/overlord.go

```diff
@@ -77,7 +77,14 @@ var (
 
 	pruneMaxChanges = 500
 
-	defaultCachedDownloads = 5
+	defaultCachePolicy = store.CachePolicy{
+		// at most this many unreferenced items
+		MaxItems: 5,
+		// unreferenced items older than 30 days are removed
+		MaxAge: 30 * 24 * time.Hour,
+		// try to keep cache < 1GB
+		MaxSizeBytes: 1 * 1024 * 1024 * 1024,
```

Comment:
what happens if a single snap >1GB ?

Reply:
This has been updated with separate policies for core and classic systems.